### PR TITLE
add some timing blocks

### PR DESCRIPTION
**Pull request by akva2**
Add  timing blocks for deck reading, state creation, state broadcasting, grid creation, grid distribution.

### CODE REVIEW

**Comment by akva2** [comment]
jenkins build this please

**Comment by hnil** [comment]
I think better coverage with profiling is good and do not see any problems here.

File: ebos/eclgenericcpgridvanguard.cc [hunk 1]
```diff
@@ -125,9 +126,12 @@ doLoadBalance_(const Dune::EdgeWeightMethod            edgeWeightsMethod,
...
         this->grid_->comm().broadcast(&loadBalancerSet, 1, 0);
 
-        const auto faceTrans = ((loadBalancerSet == 0) || partitionJacobiBlocks)
-            ? this->extractFaceTrans(gridView)
-            : std::vector<double>{};
+        std::vector<double> faceTrans;
+        {
+            OPM_TIMEBLOCK(extractTrans);
+            if (loadBalancerSet == 0 || partitionJacobiBlocks)
+                this->extractFaceTrans(gridView);
```
Comment (bska) [inline ebos/eclgenericcpgridvanguard.cc:133]: Don't you need to assign the return value to `faceTrans`?

Comment (akva2) [reply ebos/eclgenericcpgridvanguard.cc]: yes! nice catch (rebasing meh).

**Comment by akva2** [comment]
jenkins build this please

**Review by bska — APPROVED** [review]
Thanks a lot for the updates.  I'll merge into master.

**Pull request merged**